### PR TITLE
Add high-scores exercise

### LINKED ISSUE #212: Add high-scores exercise
Manage a game player's High Score list.

Your task is to build a high-score component of the classic Frogger game, one of the highest selling and addictive games of all time, and a classic of the arcade era. Your task is to write methods that return the highest score from the list, the last added score and the three highest scores.

### PR DESCRIPTION
**Pull request by josobar**
---

Fixes #212

### CODE REVIEW

File: example.jl [hunk 1]
```diff
@@ -0,0 +1,18 @@
+function list(scores)
```
Comment (SaschaMann) [inline example.jl]: This file is in the wrong dir

File: exercises/high-scores/runtests.jl [hunk 2]
```diff
@@ -0,0 +1,38 @@
+# canonical data version 4.0.0
+
+using Test
+
+include("high-scores.jl")
+
+
+@testset "List of scores" begin
+    @test [30, 50, 20, 70] == [30, 50, 20, 70]
```
Comment (SaschaMann) [inline exercises/high-scores/runtests.jl]: The canonical data suggests it should test the `scores` property. In its current form it doesn't test anything useful.

File: exercises/high-scores/example.jl [hunk 3]
```diff
@@ -0,0 +1,15 @@
...
+            return sort(a)[length(a)]
+        end
+        if sorted
+            return (sort(a)[length(a)-top+1:length(a)])
+        else
+            return reverse(sort(a)[length(a)-top+1:length(a)])
+        end
+    end
+
+    latest ? (return a[length(a)]) : return a
```
Comment (SaschaMann) [inline exercises/high-scores/example.jl]:
```suggestion
    latest ? (return a[end]) : return a
```

Comment (SaschaMann) [reply exercises/high-scores/example.jl]: You can use `end` for the last index of an array. `end + 1` and similar also work.

File: exercises/high-scores/runtests.jl [hunk 4]
```diff
@@ -0,0 +1,38 @@
+# canonical data version 4.0.0
```
Comment (SaschaMann) [inline exercises/high-scores/runtests.jl]: There are no tests for `scores` when `sorted` is given as an argument. Please add some or remove the argument.

**Comment by josobar** [comment]
Thanks for the review. I'm new to using GitHub for open-source projects like this, so the advice helps. Where does the file you mentioned was in the wrong dir belong? I will make the rest of the changes requested later.

**Comment by SaschaMann** [comment]
> Where does the file you mentioned was in the wrong dir belong?

It can be deleted, I think. You already have the correct example file in ` exercises/high-scores/example.jl `

File: exercises/high-scores/high-scores.jl [hunk 5]
```diff
@@ -0,0 +1,4 @@
+
+function scores(a::AbstractArray; latest = false, sorted=false, top=0)
```
Comment (SaschaMann) [inline exercises/high-scores/high-scores.jl]:
```suggestion
function scores(a::AbstractArray; latest = false, top=0)
```

**Comment by SaschaMann** [comment]
Could you run the notebook generator script one last time so that it reflects the change to the stub? I can't easily do it myself because I can't push to this PR's branch. Thanks.

**Comment by josobar** [comment]
This is pretty strange. I ran the generate-notebooks script again and still, this occurs.

**Comment by SaschaMann** [comment]
Are you using Windows? Did you upload it through the git CLI or the GitHub interface? In the latter case, it's possible that the line endings aren't correct.

I can generate them myself too, though it's a bit more hassle. I'll do that if you can't get it to work.

**Comment by josobar** [comment]
I am using windows and used the Git CLI.

**Comment by logankilpatrick** [comment]
Bump @josobar any way you can finish this up? Do you need help with anything here? Would be awesome to get this merged!

**Comment by cmcaine** [comment]
Closing in favour of #260 which will be easier to review.

**Pull request closed without merging**